### PR TITLE
[Docs] Instructions for new contributors to find reviewers

### PR DESCRIPTION
**Pull request by pabloantoniom**
Right now, the documentation does not clarify what new contributors should do to find potential reviewers, since new contributors don't have write access and thus cannot select them as explained in the docs. 

This patch extends the guidelines for new people, as discussed in https://discourse.llvm.org/t/how-to-find-reviewers/74803

cc: @kbeyls @DavidSpickett

### CODE REVIEW

**Review by kbeyls — APPROVED** [review]
LGTM, thank you!

I guess you may want me to merge this PR for you as you may not yet have merge rights?

**Comment by pabloantoniom** [comment]
> LGTM, thank you!
> 
> I guess you may want me to merge this PR for you as you may not yet have merge rights?

Yes please! I don't see any merge button, so I guess I don't have merge rights

File: llvm/docs/Contributing.rst [hunk 1]
```diff
@@ -98,7 +98,10 @@ To make sure the right people see your patch, please select suitable reviewers
 and add them to your patch when requesting a review. Suitable reviewers are the
 code owner (see CODE_OWNERS.txt) and other people doing work in the area your
 patch touches. Github will normally suggest some reviewers based on rules or
-people that have worked on the code before.
+people that have worked on the code before. If you are a new contributor, you
+will not be able to select reviewers in such way, in which case you can still
```
Comment (DavidSpickett) [inline llvm/docs/Contributing.rst:102]: "such a way"

Comment (pabloantoniom) [reply llvm/docs/Contributing.rst]: Done, thanks for the tip

**Pull request merged**